conda_build/build.py

```diff
@@ -1663,7 +1663,7 @@ def post_process_files(m: MetaData, initial_prefix_files):
         meta_files = (
             tuple(f for f in new_files if m.config.meta_dir in join(host_prefix, f)),
         )
-        sys.exit(
+        CondaBuildUserError(
```

Comment:
Oops missing `raise`!

```suggestion
        raise CondaBuildUserError(
```